src/app/core/entity-list/entity-list/entity-list.component.html

```diff
@@ -30,6 +30,8 @@
       [entities]="allEntities"
       [useUrlQueryParams]="true"
       [(filterObj)]="filterObj"
+      [filterString]="filterString"
+      (filterStringChange)="filterString = ''; applyFilter('')"
```

Comment:
I understand it's only resetting the string currently, but let's make it emit a new string rather than just void. That seems more intuitive to me (and may be useful in the future.